async/src/main/java/org/odk/collect/async/CoroutineAndWorkManagerScheduler.kt

```diff
@@ -25,6 +26,32 @@ class CoroutineAndWorkManagerScheduler(
         workManager
     ) // Needed for Java construction
 
+    override fun immediate(
+        tag: String,
+        spec: TaskSpec,
+        inputData: Map<String, String>,
+        notificationInfo: NotificationInfo
+    ) {
+        val workManagerInputData = Data.Builder()
+            .putString(TaskSpecWorker.DATA_TASK_SPEC_CLASS, spec.javaClass.name)
+            .putBoolean(TaskSpecWorker.FOREGROUND, true)
+            .putInt(TaskSpecWorker.FOREGROUND_TYPE, FOREGROUND_SERVICE_TYPE_DATA_SYNC)
```

Comment:
I get a warning that `FOREGROUND_SERVICE_TYPE_DATA_SYNC` was added in API 29 won't this be a problem?